docs/src/pages/WebIconsPage/IconsPage.js

```diff
@@ -35,7 +35,8 @@ import Code from '../../components/Code';
 
 const AlignedBpkSmallDownloadIcon = withButtonAlignment(BpkSmallDownloadIcon);
 
-// eslint-disable-next-line import/no-webpack-loader-syntax
+/* eslint-disable import/no-webpack-loader-syntax */
+/* eslint-disable import/no-unresolved */
```

Comment:
Same here.. Should be resolvable...

Reply:
It can be resolved when running and using it, but linting can't resolve it here